projects/igniteui-angular/src/lib/grids/filtering/grid-filtering.service.ts

```diff
@@ -67,6 +67,10 @@ export class IgxFilteringService implements OnDestroy {
         return parseInt(this.grid.parentVirtDir.getHorizontalScroll().scrollLeft, 10);
     }
 
+    public get areAllColumnsInView() {
+        return parseInt(this.grid.parentVirtDir.dc.instance._viewContainer.element.nativeElement.offsetWidth, 10) === 0;
```

Comment:
In the previous implementation you were using something else to get the view port size. Is it possible to not use private properties?

Reply:
The code is used in the quick filtering as well and I think there shouldn't be any issues